src/config.py

```diff
@@ -13,7 +15,7 @@
 
 # Utilized by imbalances_script for computing for single tx hash
 CHAIN_RPC_ENDPOINTS = {
-    "Ethereum": os.getenv("ETHEREUM_NODE_URL"),
+    "Ethereum": os.getenv("NODE_URL"),
```

Comment:
Let's decide what is this variable supposed to be? Is it one `NODE_URL` variable or one variable per chain?

Reply:
It seems that in the infra repo we define just one `NODE_URL` variable.

Reply:
So `CHAIN_RPC_ENDPOINTS` is used only when running the imbalances script for a single tx hash. When running the daemon, since it's one chain at a time, we just use `NODE_URL` and set it to being Ethereum/Gnosis/etc i.e. whatever chain we use, along with changing the `CHAIN_NAME` variable. 
Point is, `CHAIN_RPC_ENDPOINTS` is suitable for local testing. To avoid having two vars of the same value of in my .env, I changed it to NODE_URL. But it is clear for it to be ETHEREUM_NODE_URL. Will change this in the other PRs too.

Reply:
I see. Ok so let's keep the Ethereum prefix then, as the same script could be deployed for Gnosis Chain

Reply:
ok